### DISCUSSION TITLE
Moving QuickSteps not working

### DISCUSSION THREAD

**Discussion by PNMO** [microsoft/mfcmapi#512] [Q&A] [answered]
When I use Mapi to move Quick Steps from ops@precisionmovers.us (Exchange server email) to capacity@ihgoa.com (IMAP server email), using the same profile. The export and import appear to work correctly but you cannot see the QS in the menu at the top in the ribbon. I tried creating a new Outlook profile and importing there, same issue. The files move but cannot be seen in the Quicksteps box at the top of Outlook 365. Any help would be appreciated! 

Here's a video- http://somup.com/cr6rb83xmt

**Comment by Squirl1899** [comment, marked as answer]
I FINALLY figured it out!!!!

There have been SO many posts asking this same question "How do I Export Quick Steps Outlook 365 with IMAP Account using MFCMAPI" and NO one has been able to answer it.

After 5 years of trying to figure this answer out and getting SO frustrated when the only answers were pointing to this post or similar posts with the same instructions: [https://www.msoutlook.info/question/backup-and-restore-quick-steps ](https://www.msoutlook.info/question/backup-and-restore-quick-steps%C2%A0);

The issue is, ALL the guides for using MFCMAPI don't work if you are using an IMAP account.

I finally got so tired of recreating my Quick Steps (about 30) I started playing around with MFCMAPI again.

The issue with the other guides was they were pointing to the WRONG directory if you used an IMAP account.

I have put together a guide for IMAP users : https://answers.microsoft.com/en-us/outlook_com/forum/all/eureka-i-figured-it-out-export-quick-steps-outlook/cee247f8-5898-47a9-a6de-939ebda95139
